.github/workflows/test_branch.yml

```diff
@@ -0,0 +1,47 @@
+name: Test Branch
+on:
+    push:
+        branches-ignore:
+            - 'gh-pages'
+        tags-ignore:
+            - '*'
+
+jobs:
+    test-branch:
+        name: Test Branch (${{ matrix.os }})
+        runs-on: ${{ matrix.os }}
+
+        strategy:
+            matrix:
+                os: [ ubuntu-latest, windows-latest ]
+
+        steps:
+            - name: Setup Java
+              uses: actions/setup-java@v1
+              with:
+                  java-version: 8
+            - name: Setup Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: 10
+            - name: Disable line ending conversion of git
```

Comment:
Why is this necessary?

Reply:
On windows, git converts line endings from linux format to windows format. Eslint was printing a lot of warnings since we have a rule for line endings.